src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -529,10 +529,12 @@ def _get_config():
 
 @decorators.suppress_all_exceptions()
 def _get_secrets_warning_config():
-    show_secrets_warning = _get_config().getboolean('clients', 'show_secrets_warning', fallback=None)
```

Comment:
`None` is not a valid fallback value for `knack.config.getboolean`, see [code](https://github.com/microsoft/knack/blob/e0c14114aea5e4416c70a77623e403773aba73a8/knack/config.py#L144-L148):
```
    def getboolean(self, section, option, fallback=_UNSET):
        val = str(self.get(section, option, fallback))
        if val.lower() not in CLIConfig._BOOLEAN_STATES:
            raise ValueError('Not a boolean: {}'.format(val))
        return CLIConfig._BOOLEAN_STATES[val.lower()]
```
`ValueError` will be raised